<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Samplicity</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QTreeView" name="dirsTreeView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="tags" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTreeView" name="tagsTreeView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::MultiSelection</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="tagsModes" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QRadioButton" name="filterRadioButton">
            <property name="text">
             <string>Filter</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">tagMode</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="applyRadioButton">
            <property name="text">
             <string>Apply</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">tagMode</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTreeView" name="samplesTreeView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>2</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAddFile"/>
    <addaction name="actionAddDirectory"/>
    <addaction name="actionOpen_sample"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuRate">
     <property name="title">
      <string>Set Rating</string>
     </property>
     <addaction name="actionUnrated"/>
     <addaction name="actionRate_10"/>
     <addaction name="actionRate_20"/>
     <addaction name="actionRate_30"/>
     <addaction name="actionRate_40"/>
     <addaction name="actionRate_50"/>
     <addaction name="actionRate_60"/>
     <addaction name="actionRate_70"/>
     <addaction name="actionRate_80"/>
     <addaction name="actionRate_90"/>
     <addaction name="actionRate_100"/>
    </widget>
    <addaction name="actionTags"/>
    <addaction name="menuRate"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionLoop_Playback"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAddFile">
   <property name="text">
    <string>Add Files...</string>
   </property>
  </action>
  <action name="actionAddDirectory">
   <property name="text">
    <string>Add Directory...</string>
   </property>
  </action>
  <action name="actionLoop_Playback">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Loop Playback</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionTags">
   <property name="text">
    <string>&amp;Tags...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionOpen_sample">
   <property name="text">
    <string>Open Sample</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionUnrated">
   <property name="text">
    <string>Unrated</string>
   </property>
   <property name="shortcut">
    <string>`</string>
   </property>
  </action>
  <action name="actionRate_10">
   <property name="text">
    <string>Rate 10</string>
   </property>
   <property name="toolTip">
    <string>Rate 10</string>
   </property>
   <property name="shortcut">
    <string>1</string>
   </property>
  </action>
  <action name="actionRate_20">
   <property name="text">
    <string>Rate 20</string>
   </property>
   <property name="shortcut">
    <string>2</string>
   </property>
  </action>
  <action name="actionRate_30">
   <property name="text">
    <string>Rate 30</string>
   </property>
   <property name="shortcut">
    <string>3</string>
   </property>
  </action>
  <action name="actionRate_40">
   <property name="text">
    <string>Rate 40</string>
   </property>
   <property name="shortcut">
    <string>4</string>
   </property>
  </action>
  <action name="actionRate_50">
   <property name="text">
    <string>Rate 50</string>
   </property>
   <property name="shortcut">
    <string>5</string>
   </property>
  </action>
  <action name="actionRate_60">
   <property name="text">
    <string>Rate 60</string>
   </property>
   <property name="shortcut">
    <string>6</string>
   </property>
  </action>
  <action name="actionRate_70">
   <property name="text">
    <string>Rate 70</string>
   </property>
   <property name="shortcut">
    <string>7</string>
   </property>
  </action>
  <action name="actionRate_80">
   <property name="text">
    <string>Rate 80</string>
   </property>
   <property name="shortcut">
    <string>8</string>
   </property>
  </action>
  <action name="actionRate_90">
   <property name="text">
    <string>Rate 90</string>
   </property>
   <property name="shortcut">
    <string>9</string>
   </property>
  </action>
  <action name="actionRate_100">
   <property name="text">
    <string>Rate 100</string>
   </property>
   <property name="shortcut">
    <string>0</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="tagMode"/>
 </buttongroups>
</ui>
